src/logging.cc

```diff
@@ -8,10 +8,9 @@ void setupSpdlog() { std::call_once(SpdlogWasSetup, &_setupSpdlog); }
 
 void _setupSpdlog() {
   spdlog::set_default_logger(spdlog::stderr_color_mt("CHIP"));
-  spdlog::set_level(spdlog::level::debug);
   spdlog::set_pattern("%n %^%l%$ [TID %t] [%E.%F] : %v");
 
-  spdlog::level::level_enum SpdLogLevel;
```

Comment:
We should probably set the default level to "debug" if compiled in Debug and to "warn" for release ?

Reply:
fixed